src/Upgrader/Business/VersionControlSystem/GitVcs.php

```diff
@@ -35,6 +35,11 @@ class GitVcs implements VcsInterface
      */
     protected $provider;
 
+    /**
+     * @var array
```

Comment:
lets try to be more precise always please
```suggestion
     * @var array<string>
```